.github/workflows/build.yml

```
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-sbt:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scala: [ 2.11.12, 2.12.12 ]
+        spark: [ 2.4.7, 3.0.1 ]
+        exclude:
+          - scala: 2.11.12
+            spark: 3.0.1
+    name: Spark ${{matrix.spark}} on Scala ${{matrix.scala}}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.8"
+      - uses: coursier/cache-action@v5
+      - name: Build and run tests
+        run: sbt ++${{matrix.scala}} test -DSPARK_VERSION=${{matrix.spark}}
```

Now that we have to cache I'm wondering if it makes more sense to use 
```suggestion
        run: sbt clean ++${{matrix.scala}} test -DSPARK_VERSION=${{matrix.spark}}
```
?

---

Not sure. It caches downloaded sbt files only (.sbt), right?